app/client/components/user/CreateUserForm.tsx

```diff
@@ -179,30 +180,33 @@ class CreateUserForm extends React.Component<
           />
         </div>
       );
-    } else if (accountType === TransferAccountTypes.VENDOR.toLowerCase()) {
+    }
+    if (accountTypesList.includes(TransferAccountTypes.VENDOR)) {
       //  VENDOR
-      selectedAccountTypeForm = (
+      selectedVendorForm = (
         <div>
           {businessUsage}
           <InputField name="transferAccountName" label={"Store Name"} />
         </div>
       );
-    } else if (accountType === TransferAccountTypes.TOKENAGENT.toLowerCase()) {
+    }
+    if (accountTypesList.includes(TransferAccountTypes.TOKEN_AGENT)) {
       //  SUPERVENDOR
-      selectedAccountTypeForm = <></>;
+      selectedTokenAgentForm = <></>;
     }
 
     return (
       <div>
-        <ModuleHeader>Create a {accountType} account</ModuleHeader>
+        <ModuleHeader>Create an account</ModuleHeader>
 
         <div style={{ padding: "1em" }}>
           <form onSubmit={this.props.handleSubmit}>
-            <SelectField
-              name="accountType"
-              label={"Account Type"}
-              options={accountTypes}
-              hideNoneOption={true}
+            <InputField
```

Comment:
I know we can technically filter it out, but I don't think this should ever allow me to create an admin account.

Reply:
or we could just have it removed from the default i guess.